src/api/apiClients/baseApiClient.ts

```diff
@@ -0,0 +1,73 @@
+import fieldsToHideInReport from "../../data/fieldsToHideInReport.js";
+import { IRequestOptions, IResponse } from "../../types/api/apiClient.types.js";
+import { Logger } from "../../utils/logger/loggers/baseLogger.js";
+import { hideValueInObject } from "../../utils/object/index.js";
+import { BaseReporter } from "../../utils/reporter/baseReporter.js";
```

Comment:
why do not use imports from ts files? for example least types, interfaces transpile into nothing

Reply:
It is due to  "module": "ESNext" in tsconfig. Error occurs when .js is missed